docs/DeveloperGuide.md

```diff
@@ -163,6 +163,11 @@ Whenever the underlying application data is modified, the `FilteredList<Person>`
 In addition to the original `NameContainsKeywordsPredicate`, more predicates concerning each of the attributes in a `Person` are created.
 They can be fed to the `FindCommand` to filter out `Person` with the specified keywords in the specified attribute.
 
+**[Future version]** </br>
+Displaying statistics of the number of properties being sold/bought categorized by their room size to provide insight on the most popular number of rooms in a property.
+Displaying statistics of the prices of properties sold/bought to provide insight on the average property price sold/bought.
+Displaying statistics of lower price and higher price of preferences of clients to provide insight on the average asking price of a property.
+
```

Comment:
looks like you have the information for future version of Statistics parked under the "Feature `find` command" (lines 162 - 164) instead of "Statistics feature/Window" (lines 133 - 136)?

Reply:
Nice catch, thanks! I have made the change.